packages/requisitions/src/ResponseRequisition/DetailView/ContentArea.tsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
 import { DataTable, useTranslation } from '@openmsupply-client/common';
-import { useResponseColumns } from './columns';
 import { useResponseLines, ResponseLineFragment } from '../api';
 
 interface ContentAreaProps {
   onRowClick: null | ((line: ResponseLineFragment) => void);
 }
 
 export const ContentArea = ({ onRowClick }: ContentAreaProps) => {
-  const { lines, onChangeSortBy, sortBy } = useResponseLines();
-  const columns = useResponseColumns({ sortBy, onChangeSortBy });
+  const { columns, lines } = useResponseLines();
```

Comment:
not great - using columns from `useResponseColumns` won't work, after the column sort is changed the `useResponseLines` hook isn't rerendered and has the wrong sort value.
Just having another look at it..

Reply:
I give up. fewer renders like this and it works, so I'll stick with it.

Reply:
Though I am kind of diggin the columns being all wrapped up nicely with the lines 🤔